packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -213,6 +215,45 @@ class MediaQueryData {
   ///   property and how it relates to [padding] and [viewInsets].
   final EdgeInsets viewPadding;
 
+  /// The parts of the display that contain system gestures, typically swipes
+  /// from the edge of the screen for navigation.
+  ///
```

Comment:
Can you add a little more information here, e.g. that certain gestures in these areas my  not be forwarded to the app and elements placed here may therefore not respond to them.

Reply:
Sounds good, I did the following: 
```
  /// This value is typically a drag gesture from an edge of the screen.
  /// Gestures in these areas may not be forwarded to the app or may
  /// be canceled by the system. Therefore, elements placed in these areas may
  /// not properly respond to them.
```

Reply:
You might be able to reuse a version of https://github.com/flutter/flutter/pull/37416#discussion_r316366337 here.